public/pages/join-us.html

```diff
@@ -39,6 +39,7 @@
             <li><a href="/pages/programs.html">Programs</a></li>
             <li><a href="/pages/graduates-summer-2017.html">Graduates</a></li>
             <li><a href="/volunteer-opportunities">Volunteer</a></li>
+            <li><a href="/pages/software-agency.html">Software Agency </a></li>
```

Comment:
There is a space after `Agency `. If you're unsure of what the formatting should be, it is helpful to look at the surrounding code: you can note that none of the other links have a space inserted before `</a>`